day_7/fizzbuzz.rb

```diff
@@ -0,0 +1,42 @@
+# FizzBuzz App
+# version 1.0
+# by Hashim Gari
+
+
+# method that counts from 0 to given_num printing statements if matching critera
+def print_it(given_num)
+
+  # create initialize a stop variable
+  num = 0
+
+  # continously loop until num is equal to given_num
+  until num == given_num
```

Comment:
How could you change this to a range and iterate over the range? Might make it cleaner